lib/create_store.js

```diff
@@ -37,4 +35,68 @@ var createStore = function(spec) {
   return constructor;
 };
 
+// Define helper functions as properties on createStore so that they can be overridden.
+createStore.mixSpecIntoComponent = function mixSpecIntoComponent(component, spec) {
+  for (var key in spec) {
+
+    // Actions are bound, as seen.
+    if (key === "actions") {
+      if (!_isObject(spec[key])) {
+        throw new Error("Actions must be defined as an object.");
+      }
+      component.bindActions(spec[key]);
```

Comment:
Just realized, we need to chain these, if both a Store and its mixins are listening to the same action, only the Store will fire the listener.